Documentation/UpdatingToGA.md

```diff
@@ -160,6 +160,10 @@ The ClippingSphere's Radius property is now implicitly calculated based on the t
 
 The `PointerClickHandler` class has been deprecated. The `PointerHandler` should be used instead, it provides the same functionality.
 
+### HoloLens clicker support
+
+- The HoloLens clicker's controller mappings have changed from being an unhanded [`WindowsMixedRealityController`](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityController) to being an unhanded [`WindowsMixedRealityGGVHand`](xref:Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityGGVHand). To account for this, an automatic updater will run the first time you open your ControllerMapping profile. Please open your custom profile, if one exists, at least once after upgrading to GA.
```

Comment:
>Please open your custom profile, if one exists, at least once after upgrading to GA [](start = 458, length = 83)

Nit: It might be worth adding why (i.e. please open your custom profile, which will trigger a one-time migration step).

If they don't do this step, do you know if they'll still hit the null exception (i.e. in which case, can we copy that error stack or text here so that if people do forget to hit that, they might have a chance of searching for that exception here?)

Reply:
>do you know if they'll still hit the null exception

They won't. That's fixed by a different change in this PR. The only thing they'll experience is the clicker and voice sending events without mapped actions.